libshaderc_spvc/include/spvc/spvc.h

```diff
@@ -80,9 +80,43 @@ typedef enum {
 typedef enum {
   shaderc_spvc_execution_model_vertex,
   shaderc_spvc_execution_model_fragment,
-  shaderc_spvc_execution_model_glcompute
+  shaderc_spvc_execution_model_glcompute,
+  shaderc_spvc_execution_model_invalid,
 } shaderc_spvc_execution_model;
 
+typedef enum {
+  shaderc_spvc_binding_type_uniform_buffer = 0x00000000,
+  shaderc_spvc_binding_type_storage_buffer = 0x00000001,
+  shaderc_spvc_binding_type_readonly_storage_buffer = 0x00000002,
+  shaderc_spvc_binding_type_sampler = 0x00000003,
+  shaderc_spvc_binding_type_sampled_texture = 0x00000004,
+  shaderc_spvc_binding_type_storage_texture = 0x00000005,
+} shaderc_spvc_binding_type;
+
+typedef enum {
+  shaderc_spvc_texture_view_dimension_undefined = 0x00000000,
+  shaderc_spvc_texture_view_dimension_e1D = 0x00000001,
```

Comment:
Why the extra e before 1D?

Reply:
I am not sure, it is how they are named in Dawn & SPIRV-Cross, so I assuming they are from a spec somewhere.